### ISSUE TITLE
Runtime Benchmarking Guide

### ISSUE THREAD

**Issue by joepetrowski** [substrate-developer-hub/knowledgebase#4]
Add a guide to benchmarking a runtime

**Comment by athei** [comment]
@shawntabrizi started such a [documentation](https://www.shawntabrizi.com/substrate-graph-benchmarks/docs/#/). Maybe we can integrate it at some point.

**Comment by shawntabrizi** [comment]
Yes, I plan to write details about how anyone can confirm our results for Polkadot/Substrate

and generally how any runtime developer can write proper benchmarks for their systems.
